test/DynamoCoreTests/HomgeneousListTests.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using CoreNodeModels;
+using Dynamo.Events;
+using Dynamo.Graph.Nodes;
+using Dynamo.Graph.Nodes.ZeroTouch;
+using NUnit.Framework;
+
+namespace Dynamo.Tests
+{
+    [TestFixture]
+    class HomogeneousListTests : DynamoModelTestBase
+    {
+        private TimeSpan lastExecutionDuration = new TimeSpan();
+
+        protected override void GetLibrariesToPreload(List<string> libraries)
+        {
+            libraries.Add("ProtoGeometry.dll");
```

Comment:
here as well - if this difference in time required loading some number of libs, lets note them.